src/core/server/base.js

```diff
@@ -182,17 +182,18 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
       return loadOnServer({ ...renderProps, store })
         .then(() => {
           // eslint-disable-next-line global-require
-          let jedData = {};
+          let i18nData = {};
           try {
             if (locale !== langToLocale(config.get('defaultLang'))) {
               // eslint-disable-next-line global-require, import/no-dynamic-require
-              jedData = require(`json!../../locale/${locale}/${appInstanceName}.json`);
+              i18nData = require(
+                `../../locale/${locale}/${appInstanceName}.js`);
             }
           } catch (e) {
             log.info(`Locale JSON not found or required for locale: "${locale}"`);
             log.info(`Falling back to default lang: "${config.get('defaultLang')}".`);
           }
-          const i18n = new Jed(jedData);
+          const i18n = new Jed(i18nData);
```

Comment:
Setting up moment here makes the most sense to me, provided there's nothing preventing that.

```js
function makeI18n(i18nData) {
  const i18n = new Jed(i18nData);
  moment.locale(/* yadda yadda */);
  i18n.moment = moment;
  return i18n;
}

const i18n = makeI18n(i18nData);